ros/processor/inventory_events_consumer.py

```diff
@@ -105,15 +112,15 @@ def host_delete_event(self, msg):
                     f"{self.prefix} - Deleted system with inventory id: {host_id}"
                 )
 
+    @validate_ros_payload
     def host_create_update_events(self, msg):
```

Comment:
For better understanding, will it be better to have a separate handling for update & create event type?

Reply:
I think you are having this concern cause of the exception in the validator for `updated,` but we will remove that in future.
Both the engine and events consumer has two main requirements `is_ros` and `cloud_provider`, hence apart from the exception I don't see much difference w.r.t create and update from validator pov.